okio/jvm/src/main/java/okio/-Unsigned.kt

```diff
@@ -0,0 +1,380 @@
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("-Unsigned")
+@file:Suppress("NOTHING_TO_INLINE") // Syntactic sugar.
+@file:UseExperimental(ExperimentalUnsignedTypes::class)
+
+package okio
+
+import java.io.IOException
+
+/** Marker interface for APIs related to unsigned numbers. */
+@Experimental
+annotation class OkioUnsignedApi
```

Comment:
Do we want to have our own experimental annotation or pass through the `ExperimentalUnsignedTypes` annotation?

On one hand it would be nice to have our own annotation so we don't have to become non-experimental when unsigned types do, but on the other I don't see this API changing much. I'm leaning towards removing `OkioUnsignedApi`, let me know if you want me to keep it.

Reply:
I would pass it through the original because we're using their types in the public API. If we were using them as implementation detail only I would prefer our own annotation.